spec/paperclip/storage/s3_spec.rb

```diff
@@ -414,16 +425,20 @@ def original_filename
 
   context "" do
     before do
-      rebuild_model storage: :s3,
+      rebuild_model(
+        storage: :s3,
         s3_credentials: {
-        production: { bucket: "prod_bucket" },
-        development: { bucket: "dev_bucket" }
-      },
-      s3_host_alias: "something.something.com",
-      path: ":attachment/:basename:dotextension",
-      url: ":s3_alias_url"
+          production: { bucket: "prod_bucket" },
+          development: { bucket: "dev_bucket" }
+          },
```

Comment:
Indent the right brace the same as the start of the line where the left brace is.